commands.go

```diff
@@ -3349,6 +3352,38 @@ func (c cmdable) FunctionRestore(ctx context.Context, libDump string) *StringCmd
 	return cmd
 }
 
+func (c cmdable) FunctionStats(ctx context.Context) *FunctionStatsCmd {
+	cmd := NewFunctionStatsCmd(ctx, "function", "stats")
+	_ = c(ctx, cmd)
+	return cmd
+}
+
+func (c cmdable) FCall(ctx context.Context, function string, keys []string, args ...interface{}) *Cmd {
```

Comment:
It feels weird here: `client.FCall(ctx, "func", []string{"key1", "key2", "arg1", "arg2"})`

I think:
```go
 FCall(ctx context.Context, function string, keys []string, args []interface{})
```

OR

```go
type FuncCall struct {
        FuncName string
        Keys []string
        Args []interface{}
}
 FCall(ctx context.Context, f FuncCall)
```

Reply:
or: FCallArgs

Reply:
"I did not reference the eval/ro command, it is just my personal opinion.

Reply:
Of course, I was just saying I can't spot the difference between the first of the two options you suggested and the one I did.

Reply:
We cannot assume that eval/ro is correct just because it was implemented early. If there are better API implementation options in the future, the implementation of eval/ro command can also be modified.

Reply:
I agree 100%. Indeed, I'm using older implementations as a reference because I'm still learning the library, but I understand there's always space to improve :)

Reply:
~ok, then we can stick with the same implementation as eval/ro and consider improvements in the future if there are better options.